components/Ripple/examples/CardWithRippleExample.swift

```diff
@@ -64,6 +64,11 @@ class CardWithRippleExample: UIViewController {
     button.addConstraint(NSLayoutConstraint(item: button, attribute: .height, relatedBy: .equal,
                                             toItem: nil, attribute: .notAnAttribute, multiplier: 1,
                                             constant: 60))
+
+    view.accessibilityElements = [card, button]
```

Comment:
Why is this line necessary?

Reply:
yeah, otherwise the button which is a subview of the card would not be accessible